storybook/stories/API/tooltip/Tooltip.stories.tsx

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import { pageData } from '../../data';
+import { Line, LineChart, ResponsiveContainer, Tooltip } from '../../../../src';
+
+export default {
+  component: LineChart,
+};
+
+export const SimpleTooltip = {
+  render: (args: Record<string, any>) => {
+    return (
+      <ResponsiveContainer width="100%" height={400}>
+        <LineChart {...args}>
+          <Line dataKey="uv" />
+          <Tooltip />
+        </LineChart>
+      </ResponsiveContainer>
+    );
+  },
+  args: {
+    data: pageData,
+    dataKey: 'uv',
+  },
+};
+
+const CustomTooltip = ({ active, payload, label }: any) => {
+  if (active && payload) {
+    return (
+      <div
+        // className="custom-tooltip"
```

Comment:
either add these or remove them. I think they are fine to be there as classNames for an example